### DISCUSSION TITLE
MultiApp System: Having a main app with no nonlinear variables at all

### DISCUSSION THREAD

**Discussion by matthiasneuner** [idaholab/moose#21755] [Q&A General] [answered]
Hi there,

I am currently trying to set up a MOOSE multi app system, where one main app provides two sub apps with data,
but the main app itself actually should not solve anythig ( it should just move around the data). Hence,
it requires no nonlinear variables, just an aux system.

However, running a MOOSE .i file without nonlinear variables gives me
`
No variables specified in the FEProblemBase 'MOOSE Problem'`

Is there a way to have a MOOSE app which just passes around data to/from sub apps in a transient simulation?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Just add a dummy one
```
[Variables]
  [dummy] []
[]
```
OR skip the checks
```
[Problem]
  skip_nl_system_check = true
  solve = false
[]
```

**Reply by matthiasneuner** [reply]
Great, thank you !
